aboutme.md

```diff
@@ -1,20 +1,19 @@
 # Brief Introduction
 
-My name is **Ipaye Alameen**, im a computer engineering student of the university of lagos, currently in my 4th year, i love to code in Javascript and i like to think of myself as a **NERD**.
+My name is **Dunapo Amazingrace**, I'm studying Animal Science in the University Of Ibadan, currently in my 4th year, I'm an agric-tech enthusiast
 
 # Talk about Coding
 
-Well, i write **Javascript** 💗 majorly, and i strongly dislike **PHP** for now.
+Well, i write **php**  💗 majorly, I try **Javascript** sometimes too.
 
 # Favourite Emoji
 
-I really cant choose anyone right now because that would be cheating, i love them all equally but if i was to choose, id go with 🦄, you just dont see that everyday 😆.
+🤨
 
```

Comment:
Kindly delete this file and update your Pull request, it is a duplicate of what is in ` submissions/Amazingrace/aboutme.md`. Your aboutme.md should only be in your submission folder.